rsyslog/artful/master/debian/rsyslog.install

```diff
@@ -1,4 +1,5 @@
 debian/rsyslog.conf /etc/
+debian/00rsyslog.conf  usr/lib/tmpfiles.d/
```

Comment:
@wjordan Is there supposed to be a leading slash for the `usr/lib/tmpfiles.d/` path?

Reply:
The [upstream source](https://git.launchpad.net/ubuntu/+source/rsyslog/tree/debian/rsyslog.install?h=applied/ubuntu/devel#n3) does not have a leading slash- whether there is _supposed_ to be I will leave up to the maintainers of this package, but it doesn't appear to be an error not to provide one.

Reply:
@wjordan I stand corrected! My apologies for the noise in this GH issue.